backend-dotnet/Meowmentum.Server.Dotnet.Api/Meowmentum.Server.Dotnet.Business/Abstractions/IEmailService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Meowmentum.Server.Dotnet.Shared.Results;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Meowmentum.Server.Dotnet.Business.Abstractions;
+
+public interface IEmailService
+{
+    Task<Result<bool>> SendOtpByEmailAsync(string email, string otp, CancellationToken token);
```

Comment:
it is good practise to initialise cancellationtoken as default in each async method to be able to cancel task from current point but not only from root, so add this please everywhere(abstractions & implementations).